ext/com_dotnet/com_handlers.c

```diff
@@ -248,7 +248,7 @@ static PHP_FUNCTION(com_method_handler)
 	zval *object = getThis();
 	zend_string *method = EX(func)->common.function_name;
 	zval *args = NULL;
-	php_com_dotnet_object *obj = (php_com_dotnet_object*)object;
+	php_com_dotnet_object *obj = (php_com_dotnet_object*)CDNO_FETCH(object);
```

Comment:
The cast is superfluous, since [`CDNO_FETCH`](https://github.com/php/php-src/blob/53d8fa5af71e91799402224d4a137c2ee759a187/ext/com_dotnet/php_com_dotnet_internal.h#L63) already does that cast.

Reply:
@cmb69 yeah, I see. Thanks for the fixes.